doc/classes/@GDScript.xml

```diff
@@ -673,27 +760,48 @@
 			</return>
 			<description>
 				Return a random floating point value between 0 and 1.
+				[codeblock]
+				# prints 0.375671
+				print(randf())
+				[/codeblock]
 			</description>
 		</method>
 		<method name="randi">
 			<return type="int">
 			</return>
 			<description>
-				Return a random 32 bits integer value. To obtain a random value between 0 to N (where N is smaller than 2^32 - 1), you can use remainder. For example, to get a random integer between 0 and 19 inclusive, you can use randi() % 20.
+				Return a random 32 bits integer value. To obtain a random value between 0 to N (where N is smaller than 2^32 - 1), you can use remainder.
+				[codeblock]
+				print(randi() % 20)       # prints random number from 0 - 19
+				print(randi() % 100)      # prints random number from 0 - 99
+				print(randi() % 100 + 1)  # prints random number from 1 - 100
+				[/codeblock]
 			</description>
 		</method>
 		<method name="randomize">
 			<return type="void">
 			</return>
 			<description>
 				Randomize the seed (or the internal state) of the random number generator. Current implementation reseeds using a number based on time.
+				[codeblock]
+				func _ready():
+				    randomize()
+				[/codeblock]
 			</description>
 		</method>
 		<method name="range" qualifiers="vararg">
 			<return type="Array">
 			</return>
 			<description>
 				Return an array with the given range. Range can be 1 argument N (0 to N-1), two arguments (initial, final-1) or three arguments (initial, final-1, increment).
+				[codeblock]
+				# prints (0, 1)
+				print(range(2))
+				# prints (2, 3)
+				print(range(2, 4))
+				# prints [2, 4, 6, 8]
+				print(range(2, 10, 2))
```

Comment:
I think `range` is designed for being iterated. Maybe examples should be more of `for i in range(...`?